logrus.go

```diff
@@ -40,12 +38,12 @@ type MozLogFormatter struct {
 }
 
 func (m *MozLogFormatter) Format(entry *logrus.Entry) ([]byte, error) {
-	appLog := &mozlog.AppLog{
+	appLog := &AppLog{
 		Timestamp:  entry.Time.UTC().UnixNano(),
 		Time:       entry.Time.UTC().Format(time.RFC3339),
 		Type:       "app.log",
 		Logger:     m.LoggerName,
-		Hostname:   mozlog.Hostname(),
+		Hostname:   Hostname(),
 		EnvVersion: "2.0",
 		Pid:        os.Getpid(),
```

Comment:
should probably cache is variable as well to avoid the system call.

Reply:
It calls https://github.com/mozilla-services/go-mozlogrus/pull/6/files#diff-e49fa914308349fb61e3d6ef6b8a0fdeR10 which caches the hostname.

Reply:
I could also add it the hostname caching to an init instead.